client/src/client.rs

```diff
@@ -919,6 +936,15 @@ pub mod events_api {
             }
         }
 
+        fn decode_publisher_message(bytes: &[u8]) -> Result<VersionedEventPublisherMessage> {
+            let mut res = VersionedEventPublisherMessage::decode_all_versioned(bytes);
+            if res.is_err() {
+                warn!("Can't decode event publisher message using all bytes");
```

Comment:
```suggestion
                warn!("Can't decode event publisher message. Not all bytes were consumed");
```

Reply:
You are suggesting "Failed to..." on line 733, I'd say use the same structure here as well

Reply:
I'm agree with @outoftardis , using `.` or `:` in error messages could make message unreadable when errors are chained